src/components/Button/Button.scss

```diff
@@ -0,0 +1,25 @@
+@use "../../styles/variables" as *;
+@use "../../styles/typography" as *;
+@use "../../styles/reset" as *;
+
+.button {
+  background-color: $background-color;
+  width: 9rem;
+  height: 3rem;
+  font-size: 1rem;
+  padding: 10px 20px;
+  border: solid black;
+}
+
+.button-enter {
```

Comment:
`-enter` is a modifier so the class should be `button--enter`

Reply:
changed